edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -43,8 +43,6 @@ public static class Constants
 
         public const string EdgeModuleHubServerCertificateFileKey = "EdgeModuleHubServerCertificateFile";
 
-        public const string CheckImagePullBeforeModuleCreate = "CheckImagePullBeforeModuleCreate";
```

Comment:
Want to call out this is being removed.

Reply:
I think it should not be removed for case when customer might have been using this to false and they want to use the old functionality

Reply:
I think it is safe to remove, and a good idea to remove given that:
1. Hasn't been exposed to customers by any documentation
2. Was added as a failsafe to prevent a double regression in case we weren't sure if it would lead to new problems. It has been running now for quite some time so I think it is better to remove it.

Reply:
We can discuss offline in our scheduled meeting for today